karma.conf.js

```diff
@@ -33,18 +33,18 @@ if (coverageEnabled) {
 module.exports = (config) => {
     const webpackConfig = require('./webpack.dev.js');
     delete webpackConfig.output;
-    if (coverageEnabled) {
```

Comment:
if i'm reading this right, we need to remove all references to karma instrumentation, including any npm scripts

Reply:
still left the karma istanbul reporter, as I assume it reads the instanbul output from babel-plugin-istanbul. Basically we only swapped istanbul-instrumenter-loader for babel-plugin-istanbul, but the rest stays the same as far as Karma, I believe.